### PR TITLE
Feature: [Review Sort] Buttons

### PR DESCRIPTION
**Pull request by dterr**
Create buttons for Sorting Reviews.
Buttons for:
- Sort by recency
- Sort by relevancy
Add CSS styling for buttons and their button group
Will be wired with JS sorting scripts hand off from JavaServlet

### CODE REVIEW

**Comment by dterr** [comment]
This will be pushed to a larger feature branch.

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -45,6 +45,10 @@
         <div class="modal-header"> 
           <div class="modal-title">Results</div>
           <button id="modal-backarrow"></button>
+          <div class="button-group">
+            <button> Recent </button>
```
Comment (ccjeane) [inline src/main/webapp/index.html:49]: Remove spaces before and after Recent / Relevant as per html style guidelines!

File: src/main/webapp/modal-overlay.css [hunk 2]
```diff
@@ -151,6 +151,33 @@ a:hover {
   color: rgba(230, 230, 230); /* Grey. */ 
 }
 
+.button-group button {
+  cursor: pointer;
+  padding: 10px 24px;
```
Comment (ccjeane) [inline src/main/webapp/modal-overlay.css:156]: CSS should be alphabetical!

File: src/main/webapp/index.html (hunk 1 above)
Comment (dterr) [reply src/main/webapp/index.html:49]: Dangit I remebered there was something I had to do with it I just forgot what it was.

File: src/main/webapp/modal-overlay.css (hunk 2 above)
Comment (dterr) [reply src/main/webapp/modal-overlay.css:156]: Shoot! I made the comment on a previous PR. I should have been more careful.

**Pull request merged**